Library/Formula/rust.rb

```diff
@@ -44,7 +44,6 @@ def install
     args = ["--prefix=#{prefix}"]
     args << "--disable-rpath" if build.head?
     args << "--enable-clang" if ENV.compiler == :clang
-    args << "--release-channel=beta" unless build.head?
```

Comment:
Should this not say `--release-channel=stable`? It looks like the default channel is "dev" even in the 1.0.0 tarball.